lisa/messages.py

```diff
@@ -137,3 +135,65 @@ class NetworkPPSPerformanceMessage(PerfMessage):
     rx_tx_pps_minimum: Decimal = Decimal(0)
     rx_tx_pps_average: Decimal = Decimal(0)
     rx_tx_pps_maximum: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkTCPPerformanceMessage(PerfMessage):
+    tool: str = constants.NETWORK_PERFORMANCE_TOOL_NTTTCP
```

Comment:
leave it empty to prevent incorrect put to ntttcp.

Reply:
updated